src/command/helpModule.ts

```diff
@@ -39,9 +40,23 @@ export default class HelpModule extends Module {
                 true
             );
         }, initialEmbed);
-
-        if (!msg.guild.me.permissionsIn(msg.channel).has("EMBED_LINKS")) {
-            await msg.channel.send(":warning: Can't send embeds to this channel!")
+        
+        const nonEmbed = `${CODEBLOCK}
+${Array.from(this.client.modules)
+    .filter(mod => getModuleCommands(mod).length !== 0)
+    .map(
+        module => `${module.constructor.name}:
+${Array.from(this.client.commandManager.cmds)
+    .filter(c => c.module == module)
+    .map(cmd => `	${prefix}${cmd.id.split("/")[1]}`)
+    .join("\n")}`
+    )
+    .join("\n")}
+${CODEBLOCK}`
```

Comment:
Maybe we can move the .maps and .filters outside of the quotes to clean this up a little?

Reply:
just took this from the code before the embeds, just thought I'd keep it how it was.

Reply:
I agree with @webmail - this is quite a mess (and not even that readable, took me a while to realize that there's nested template strings, yes I know it's my code but still)